releasenotes/notes/unify-instructions-and-commands-aaa6d8724b8a29d3.yaml

```diff
@@ -25,9 +27,21 @@ features:
 
         sched += Delay(5, DriveChannel(0))
         sched += ShiftPhase(np.pi, DriveChannel(0))
-        sched += Acquire(100, AcquireChannel(0), MemorySlot(0))
+        sched += Play(SamplePulse([1.0, ...]), DriveChannel(0))
         sched += SetFrequency(5.5, DriveChannel(0))  # New instruction!
+        sched += Acquire(100, AcquireChannel(0), MemorySlot(0))
+  - |
+    There is now a :py:class:`~qiskit.pulse.instructions.Play` instruction
+    which takes a description of a pulse envelope and a channel. The pulse
+    description must subclass from :py:class:`~qiskit.pulse.pulse_lib.Pulse`.
+    py:class:`~qiskit.pulse.pulse_lib.SamplePulse` and
+    py:class:`~qiskit.pulse.pulse_lib.ParametricPulse` s (e.g. ``Gaussian``)
+    now subclass from ``Pulse`` and have been moved to the ``pulse_lib``.
```

Comment:
This was a bit hard to follow, Originally I thought this should have been the pulse description should either be `Pulse`, `SamplePulse`, or `ParametricPulse`. But then I realized what it was actually saying. Maybe we should split this into 2 release notes. One saying that `SamplePulse` and `ParametricPulse` are now subclasses of `Pulse` and have been moved to `pulse_lib` (maybe as an upgrade note?) and the other about `Play`.

Reply:
This is a good idea, thank you!